classes/person.rb

```diff
@@ -10,6 +10,12 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.person = self
```

Comment:
- [ ] The same thing goes for the **person.rb**  here is a hint  about the class person ` The code is a class that has three attributes: name, age, and rentals.
 The initialize method is used to create an instance of the class with the given values for those attributes.
 The add_rental method creates a new rental object using the date as its key and the book as its value.
 The code creates a new class called Person.
 The code also defines the following attributes: name, age, and rentals.`